src/terraform/index_test.go

```diff
@@ -69,7 +69,7 @@ provider "aws" {
 }
 
 module "aws" {
-  source = "./aws"
+  source = "git@github.com:Originate/exosphere.git//src//terraform//modules//aws?ref=8786f912"
```

Comment:
Why the `//` for the file? Also should we specify a commit or anything? Currently this is pulling from latest master always